tests/test_app/migrations/0002_testmodelnestedinline.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10 on 2018-04-30 12:39
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('test_app', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='TestModelNestedInline',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('nested_inline_name', models.CharField(default=b'v1', max_length=191)),
+                ('test_model_inline', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='test_app.TestModelInline')),
+            ],
+        ),
+    ]
```

Comment:
Can you merge this migration with the 0001_initial migration for the test app?

There's no reason to proliferate migration files for test databases.